docs/public/static/schemas/unversioned/eas-json-build-common-schema.js

```diff
@@ -1,4 +1,11 @@
 export default [
+  {
+    name: 'withoutCredentials',
+    type: 'boolean',
+    description: [
+      "When set to `true`, EAS CLI won't require you to configure credentials when building the app. This comes in handy when using EAS Build custom builds",
```

Comment:
```suggestion
      "When set to `true`, EAS CLI won't require you to configure credentials when building the app. This comes in handy when using EAS Build [custom builds](/custom-builds/get-started/).",
```

Let's also add an internal link here.